src/metabase/server/middleware/offset_paging.clj

```diff
@@ -21,9 +21,16 @@
   Automatically generated by a handler in offset-paging middleware."
   false)
 
+(defn- ensure-single-value
+  "For parameters that should have a single value, only return the 1st if multiple values were passed in the request"
+  [v]
+  (if (vector? v)
```

Comment:
This is all reasonable, but I'm not convinced that the collections of params will always be vectors. Is this too concrete of a collection type? Should it be something more generally "sequential"-y?

Reply:
Good point. New version look better?